using_the_compiler/README.md

```diff
@@ -44,6 +44,12 @@ Make sure to always use `--release` for production-ready executables and when pe
 
 The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the `crystal` command almost as if it were an interpreter.
 
+## Reducing the executable's binary size
+
+This will remove debug symbols. Use this options only if the size is an issue and debugging isn't planned.
+
+To reduce the binary size, the `--no-debug` flag can be added. Furthermore, commands like `strip` can also be used.
```

Comment:
The reference to strip is a bit too loose. Is it a flag for the compiler? Is it the Unix `strip` tool? Either remove the reference or add a bit more context assuming the reader has never been exposed to `strip`.